src/web/src/app/app.component.html

```diff
@@ -46,19 +46,20 @@ <h1 class="branding__text" *ngIf="!mobileQuery.matches">SvetIT</h1>
         </mat-select>
       </button>
     </div>
-
-    <!-- a routerLink="/user" -->
-    <div class="app-toolbar__user app-toolbar__user-link" (click)="toggleDropDown()">
+  </ng-container>
+  <ng-container *ngIf="authorized">
+	<div class="app-toolbar__user app-toolbar__user-link" (click)="toggleDropDown()">
       <mat-icon class="app-toolbar__user-icon">person</mat-icon>
       <span class="app-toolbar__user-name" *ngIf="!mobileQuery.matches">
         <strong>{{auth.user.username}}</strong>
       </span>
       <div class="dropdown-user-menu" [class.force-visible]="showDropDown">
-        <a routerLink="/user" i18n="@@PROFILE">Профиль</a>
-        <a routerLink="/user/scheme-groups" i18n="@@SCHEME_GROUPS.GROUPS">Группы</a>
+        <a [ngClass]="hasWspace ? '': 'inactiveLink'" routerLink="/user" i18n="@@PROFILE" inactiveLink>Профиль</a>
+        <a [ngClass]="hasWspace ? '': 'inactiveLink'" routerLink="/user/scheme-groups" i18n="@@SCHEME_GROUPS.GROUPS" inactiveLink>Группы</a>
```

Comment:
А для чего тут в конце тега inactiveLink, вроде в HTML нет такого аттрибута? Как это работает?
И кажется можно лепить css классы к тегу так как в 56 строке сделано:
```suggestion
        <a [class.inactiveLink]="hasWspace" routerLink="/user" i18n="@@PROFILE">Профиль</a>
        <a [class.inactiveLink]="hasWspace" routerLink="/user/scheme-groups" i18n="@@SCHEME_GROUPS.GROUPS">Группы</a>
```

Reply:
Извиняюсь, не доглядел. Сначала хотел через кастомные атрибуты сделать, потом забыл этот inactiveLink убрать.

По варианту добавления классов, как в 56 строке, пока не разобрался. Если текущая реализация в целом устраивает, то можно её и оставить.

Reply:
Установку классов поменял на вариант, предложенный тобой. Работает.